.github/workflows/reuseable_publish.yaml

```diff
@@ -48,6 +48,7 @@ jobs:
           sourceDirectory: /tmp/sovtoken-deb/
           distribution: ${{ inputs.distribution }}
           component: ${{ inputs.REPO_COMPONENT }}
+          repository: "deb"
```

Comment:
The action defines this input parameter as `repo`: https://github.com/pSchlarb/indy-shared-gha/blob/63617fae9e52f630c034c0444b2d41adfb09d9ef/.github/actions/publish-deb/action.yaml#L16-L19